kwave/ktransducer.py

```diff
@@ -644,7 +644,7 @@ def delay_mask(self, mode=None):
             ].min()  # -1s compatibility
         else:
             mask[unflatten_matlab_mask(mask, active_elements_index - 1)] += self.stored_beamforming_delays_offset  # -1s compatibility
-        return mask.astype(np.uint8)
```

Comment:
Again here for one of the operations, the unit8 type was not being correctly promoted for values larger than 255 so I bumped the type.